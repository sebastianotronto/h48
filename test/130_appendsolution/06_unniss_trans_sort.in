B (R2 F2)
1
1
rotation UR
0
