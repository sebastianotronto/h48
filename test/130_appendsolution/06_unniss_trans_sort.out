R2 L B2
Number of solutions: 1
Shortest solution length: 3
Used bytes: 8
